developer_guide/migration/index.rst

```diff
@@ -100,11 +100,12 @@ Here are the migration guides:
 
 **Enterprise Edition**
 
-Please contact our team through your access to the support service desk to ask for a standard archive.
+Standard Enterprise Edition (EE) archives are now available on a dedicated portal: https://partners.akeneo.com/. You will be able to download your EE archive directly from it. If you do not have access to this interface, please contact your Sales or Channel Manager.
```

Comment:
please contact your **Customer Success** or Channel Manager (instead of Sales Manager)